src/components/marquee-wizard/marquee-wizard.component.spec.ts

```diff
@@ -571,3 +571,151 @@ describe('Marquee wizard with custom step template', () => {
         expect(steps.item(1).textContent.trim()).toBe('1. Step Two (456)');
     });
 });
+
+@Component({
+    selector: 'marquee-wizard-step-navigation-app',
+    template: `
+        <ux-marquee-wizard
+            [(step)]="currentStep"
+            [sequential]="sequential">
+            <ux-marquee-wizard-step
+                header="Step One"
+                [(visited)]="step1Visited"
+                [valid]="step1Valid"
+                [(completed)]="step1Completed">
+                Step One Content
+            </ux-marquee-wizard-step>
+            <ux-marquee-wizard-step
+                header="Step Two"
+                [(visited)]="step2Visited"
+                [valid]="step2Valid"
+                [(completed)]="step2Completed">
+                Step Two Content
+            </ux-marquee-wizard-step>
+            <ux-marquee-wizard-step
+                header="Step Three"
+                [(visited)]="step3Visited"
+                [valid]="step3Valid"
+                [(completed)]="step3Completed">
+                Step Three Content
+            </ux-marquee-wizard-step>
+        </ux-marquee-wizard>
+    `
+})
+export class MarqueeWizardStepNavigationComponent {
+    currentStep: number;
+    sequential: boolean = true;
+
+    // step 1 values
+    step1Valid: boolean = true;
+    step1Visited: boolean = true;
+    step1Completed: boolean = false;
+
+    // step 2 values
+    step2Valid: boolean = true;
+    step2Visited: boolean = true;
+    step2Completed: boolean = false;
+
+    // step 3 values
+    step3Valid: boolean = true;
+    step3Visited: boolean = true;
+    step3Completed: boolean = false;
+
+}
+
+describe('Marquee wizard with step navigation', () => {
+    let component: MarqueeWizardStepNavigationComponent;
+    let fixture: ComponentFixture<MarqueeWizardStepNavigationComponent>;
+    let nativeElement: HTMLElement;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            imports: [MarqueeWizardModule],
+            declarations: [MarqueeWizardStepNavigationComponent]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(MarqueeWizardStepNavigationComponent);
+        component = fixture.componentInstance;
+        nativeElement = fixture.nativeElement;
+        fixture.detectChanges();
+    });
+
+    describe('[sequential]=false', () => {
+        it('should allow navigation to any step', async () => {
```

Comment:
Could we add some tests to these to cover outputs being emitted, for example, check stepChange, visitedChange and completedChange on the steps are emitted the correct number of times with the correct values,